dev/com.ibm.ws.org.apache.wss4j.ws.security.stax.2.3.0/src/org/apache/wss4j/stax/setup/WSSec.java

```diff
@@ -52,14 +52,26 @@
  * Instances of the inbound and outbound security streams can be retrieved
  * with this class.
  */
-//No Liberty code change
 public class WSSec {
 
     //todo outgoing client setup per policy
+    private static final org.slf4j.Logger LOG =
```

Comment:
This totally seems like incorrect code.  Why would you need to do this?

Reply:
I will look in to this more. I made this change, so we can workaround an issue (#17758)